### ISSUE TITLE
option to not edit the  Date Added? 

### ISSUE THREAD

**Issue by suruoxi** [AllanChain/zotero-arxiv-workflow#27]
when the arXiv paper status updated, the  Date Added is refreshed too.  This may cause inconveniences when sort items by Date Added.

**Comment by AllanChain** [comment]
Thanks for reporting. This is unexpected and should be treated as a bug. However, I'm having problems to reproduce it. Can you provide more details?

**Merging items**. Before:
![image](https://github.com/user-attachments/assets/eb6a00dc-0493-4a42-8ae1-db2fdcd71bb2)
After:
![image](https://github.com/user-attachments/assets/97e961b3-a65e-4e19-a44d-70856356f345)

**Updating arXiv version**. Before:
![image](https://github.com/user-attachments/assets/6e73674e-96c3-463e-bee5-1e324390f776)
After:
![image](https://github.com/user-attachments/assets/87ff6e66-c7f7-487f-a91f-4885b39e8c70)

**Comment by suruoxi** [comment]
I verified again and found that this bug indeed does not exist. 

When I submitted the issue, I was working with a very large collection, during the updating of arxiv paper, there was a moment when both the published version and the arxiv version coexisted. I probably only saw the timestamp of the official version. Actually, the two versions will be merged later. The merged timestamp is correct. 

Sorry for the confusion.
